i18n/es.json

```diff
@@ -28,7 +28,9 @@
   "UIKit.MoneyField.highPrecision": "",
   "UIKit.MultilineTextInput.collapse": "Contraer",
   "UIKit.MultilineTextInput.expand": "Expandir",
+  "UIKit.PasswordField.hide": "",
+  "UIKit.PasswordField.show": "",
```

Comment:
@qmateub can you provide the spanish translations?

Reply:
hide: ocultar
show: demostrar

To be frank: we're not following any translation process in doing this.

Reply:
@tdeekens we should open a JIRA ticket for these translations JSYK

Reply:
JSYK i know that's why i stated that.

Reply:
I think "revelar" is is a better spanish word than "demonstrar" for this context.

But so should we really set these translations here?

Reply:
I don't know how strict we want to be. @emmenko suggested it. Maybe ask Camilo? Maybe Maria can give Spanish in this case? Given it is intended to go through the process we need to update/pull these later.

Reply:
I'll check with Camilo. We should provide the translations before merging this though.

Reply:
Okay. I asked Maria. I also did a quick search and seems like "mostrar" (contraseña) seems to be widely used.